projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -164,15 +151,16 @@ export class CommentComponent implements OnChanges, OnDestroy {
       };
       this.updated.emit(payload);
       this.hasUnsavedChanges = false;
-      this.editable = false;
+      this._editable = false;
       this.changes.emit(false);
     }
   }
 
   onCommentClick() {
     if (!this.selected) {
       this.selected = true;
-      this.commentClick.emit({ annotationId: this._comment.annotationId, editable: this._editable });
+      this._editable = false;
+      this.commentClick.emit({ annotationId: this._comment.annotationId, editable: this._editable, selected: true });
```

Comment:
not sure how this works with the state, in terms of managing the selection

Reply:
have explained verbally